specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/credential.json

```diff
@@ -0,0 +1,283 @@
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "credentials"
+        ],
+        "operationId": "Credential_Delete",
+        "x-ms-examples": {
+          "Credentials_Delete": {
+            "$ref": "./examples/Credentials_Delete.json"
+          }
+        },
+        "description": "Deletes a credential.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/credentialName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "202": {
+            "description": "Accepted."
+          },
+          "204": {
+            "description": "No Content."
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CredentialListResponse": {
+      "description": "A list of credential resources.",
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of credentials.",
+          "items": {
+            "$ref": "#/definitions/ManagedIdentityCredentialResource"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of results, if any remaining results exist.",
+          "type": "string"
+        }
+      }
+    },
+    "ManagedIdentityCredentialResource": {
+      "description": "Credential resource type.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "artifacts.json#/definitions/SubResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "./entityTypes/Credential.json#/definitions/ManagedIdentityCredential",
+          "description": "Managed Identity Credential properties."
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    }
+  },
+  "parameters": {
+    "workspaceName": {
+      "name": "workspaceName",
+      "description": "The workspace name.",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The Synapse client API Version.",
+      "x-ms-parameter-location": "method"
+    },
+    "endpoint": {
+      "name": "endpoint",
+      "description": "The workspace development endpoint, for example https://myworkspace.dev.azuresynapse.net.",
```

Comment:
please refer to this PR to fix the potential broken links https://github.com/Azure/azure-rest-api-specs/pull/26143